tcad_bot/handlers.py

```diff
@@ -30,8 +30,9 @@ def get_uv_index(bot, update):
         filter(lambda p: p['nombre'] == 'SANTIAGO', radiation_data)
     )
 
+    text = 'Pronostico ' + radiation_stgo['fechapron'] + ': ' + radiation_stgo['indicepron']
     bot.sendMessage(
-        chat_id=update.message.chat_id, text=radiation_stgo['indiceobs'])
+        chat_id=update.message.chat_id, text)
```

Comment:
You can't pass a positional argument after a keyword argument. Either delete `chat_id=` or add `text=text`.